pkg/database/database.go

```diff
@@ -713,3 +716,26 @@ func uintPtr(v uint) *uint {
 func uintDiff(then, now uint) string {
 	return stringDiff(strconv.FormatUint(uint64(then), 10), strconv.FormatUint(uint64(now), 10))
 }
+
+func initialHMAC(keys []envconfig.Base64Bytes, data string) (string, error) {
```

Comment:
Hmm - why couple these to envconfig?

Reply:
discussed live - added comment, keeping types.